rousseau_vote/lib/src/widgets/rousseau_logged_scaffold.dart

```diff
@@ -0,0 +1,30 @@
+
+
+import 'package:flutter/material.dart';
+import '../config/app_constants.dart' as AppConstants;
+import 'logged_screen.dart';
+
+// Widget to use for loggedin screens. It checks if the user is logged in. If
+// not it redirects to the login screen. It also provide the default rousseau
+// toolbar
+class RousseauLoggedScaffold extends StatelessWidget {
+  final Widget body;
+  const RousseauLoggedScaffold(this.body);
+
+  @override
+  Widget build(BuildContext context) {
+    return LoggedScreen(
+        Scaffold(
+          appBar: AppBar(title: Text(AppConstants.TOOLBAR_TITLE)),
```

Comment:
Shouldn't we give it an optional argument to allow us to override the default title if needed

Reply:
Yes we can do that. I'd add it as far as we need it. We just need to understand how many type of screens we have.